baselines/Common Controls Minimum Viable Secure Configuration Baseline v0.1.md

```diff
@@ -1029,57 +939,20 @@ To add an app to the allowlist:
 
 3.  Click **Allowlisted Apps** to manage the allow list.
 
-## 13. Less Secure Apps
-
-This control disables legacy authentication and requires the use of modern authentication protocols based on federation for access from applications.
-
-Some older versions of common software may break when this control is implemented. Examples of these apps include:
-
--   Mails configured with POP3
--   Older versions of Outlook
```

Comment:
I'd recommend working these details into the description for the "Authorized Google Marketplace Apps" group.

Reply:
@adhilto Added.